crates/core/app/tests/mock_consensus.rs

```diff
@@ -98,6 +98,7 @@ async fn mock_consensus_can_send_a_sequence_of_empty_blocks() -> anyhow::Result<
 }
 
 #[tokio::test]
+#[ignore]
```

Comment:
do not merge this kate!! (_this was helpful when working in a `cargo watch` loop_)

Reply:
9f569c4a0